src/config/htmlToSlate/types.ts

```diff
@@ -4,21 +4,32 @@ interface ItagMap {
   [key: string]: (a?: Element) => object
 }
 
+/**
```

Comment:
Although docs are included, I find it easier to revisit the configuration object when appropriate comments are included to summarise each option!